src/stylesheets/core/mixins/_screen-reader.scss

```diff
@@ -6,9 +6,7 @@
 }
 
 @mixin add-sr-only {
```

Comment:
Seemed like a duplicate of `sr-only` mixin so I extended it instead.

Reply:
good idea